ilastik/widgets/featureTableWidget.py

```diff
@@ -182,8 +182,8 @@ def setIconAndTextColor(self, color):
 # ItemDelegate
 # ==============================================================================
 class ItemDelegate(QItemDelegate):
-    """"
-     TODO: DOKU
+    """ "
```

Comment:
stray `"`

Reply:
Okay I took care of that one and also re-applied black on current master; force-pushed here.